<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>438</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>644</width>
    <height>438</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>644</width>
    <height>438</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>zoffline_launcher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_main">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_options">
       <property name="spacing">
        <number>16</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_status">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <layout class="QFormLayout" name="formLayout_zofflineStatus">
           <property name="labelAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_zofflineStatusTip">
             <property name="text">
              <string>Zoffline状态:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_zofflineStatus">
             <property name="styleSheet">
              <string notr="true">QLabel { color: rgb(170, 85, 0); }</string>
             </property>
             <property name="text">
              <string>加载中</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_zwiftStatus">
           <property name="labelAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_zwiftStatusTips">
             <property name="text">
              <string>Zwift状态:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_zwiftStatus">
             <property name="styleSheet">
              <string notr="true">QLabel { color: rgb(255, 0, 0); }</string>
             </property>
             <property name="text">
              <string>未安装</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_config">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <layout class="QFormLayout" name="formLayout_connectMethod">
           <property name="labelAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBoxconnectMethod"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_connectMethodTip">
             <property name="text">
              <string>连接方式：</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_customIp">
           <property name="labelAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_customIpTip">
             <property name="text">
              <string>自定义服务器IP地址：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditcustomIp"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_launchZwift">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>启动Zwift</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser_log"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
